doc/rtd/topics/datasources/ec2.rst

```diff
@@ -102,4 +113,11 @@ Notes
    The check for the instance type is performed by is_classic_instance()
    method.
 
+ * For EC2 instances with multiple network interfaces (NICs) attached, dhcp4
+   will be enabled to obtain the primary private IPv4 address of those NICs.
+   Wherever dhcp4 or dhcp6 is enabled for a NIC, a dhcp route-metric will be
+   added with the value of ``<device-number + 1> * 100`` to ensure dhcp
+   routes on the primary NIC are preferred to any secondary NICs.
+   For example: eth0 will have a DHCP route-metric of 100, eth1 will be 200.
```

Comment:
We might want to steer away from interface names here, because the kernel and the IMDS may disagree about the ordering of interfaces, and (I think?) we will use MAC addresses to ensure that the IMDS' naming wins. Am I being too cautious?

Reply:
I think that's a reasonable level of caution.
Now reads:
```
  For example: the primary NIC will have a DHCP route-metric of 100,
  the next NIC will be 200.
```